tests/playwright-test/reporter-raw.spec.ts

```diff
@@ -110,6 +113,47 @@ test('should save attachments', async ({ runInlineTest }, testInfo) => {
   expect(path2).toBe('dummy-path');
 });
 
+test.extend<{ tmpDir: string }>({
```

Comment:
Why do we need this fixture? Might as well just create a file in the test, and then remove it manually?

Reply:
What if I write a buggy test or it gets interrupted while executing? Then I litter the FS with temp files I haven't removed! Playwright Test fixtures and hooks ensure these get cleaned up properly! 😄 

Let me know if that makes sense, otherwise, I can remove (since it would most likely get removed or worst case leave behind a temp file on the host machine).

Reply:
If you create a file inside `TestInfo.outputPath(...)` it will not be left behind :)